src/components/Credit/index.jsx

```diff
@@ -129,6 +129,9 @@ function Credit({ shakeCredit, register, setRegister, creditRef }) {
             </div>
           )}
         </button>
+        <p className="text-slate-600 font-medium text-sm text-center mt-7">
```

Comment:
```suggestion
        <p className="text-darkgray font-medium text-sm text-center mt-7">
```

피그마랑 컬러 값이 다른데, 확인 부탁할게!

Reply:
아 확인했어! 다르구나..ㅠㅠ 다음에는 더 신경써서 작업할게!